### PR TITLE
Remove redundant public for interface methods

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Removed redundant `public` modifier for interface methods.  Note some of the file already had some methods without the `public` so this makes things consistent.

### CODE REVIEW

File: src/main/java/org/isf/visits/service/VisitsIoOperationRepository.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	List<Visit> findAllWhereWardByOrderPatientAndDateAsc(@Param("wardId") String wardId);
 
 	@Query(value = "SELECT * FROM VISITS WHERE VST_WARD_ID = :ward ORDER BY VST_DATE", nativeQuery= true)
-	public List<Visit> findAllWhereWardByOrderDateAsc(@Param("ward") String ward);
+    List<Visit> findAllWhereWardByOrderDateAsc(@Param("ward") String ward);
```
Comment (mwithi) [inline src/main/java/org/isf/visits/service/VisitsIoOperationRepository.java:24]: could please you use the https://github.com/informatici/openhospital-core/blob/develop/.editorconfig included in the project?

File: src/main/java/org/isf/dicom/manager/DicomManagerInterface.java [hunk 2]
```diff
@@ -34,7 +34,7 @@
     * @return true if file exist
      * @throws OHServiceException 
     */
```
Comment (mwithi) [inline src/main/java/org/isf/dicom/manager/DicomManagerInterface.java:36]: May we ask you to fix this indentation?

File: src/main/java/org/isf/dicom/manager/DicomManagerInterface.java [hunk 3]
```diff
@@ -43,7 +43,7 @@
      * @return true if file exist
       * @throws OHServiceException 
      */
```
Comment (mwithi) [inline src/main/java/org/isf/dicom/manager/DicomManagerInterface.java:45]: May we ask you to fix this indentation?

File: src/main/java/org/isf/visits/service/VisitsIoOperationRepository.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/visits/service/VisitsIoOperationRepository.java:24]: I am using .editorconfig and Intellij and I am not seeing the indentation issue that is showing up here in the diffs.  I am assuming it has to do with "tab size"; that is why I much prefer only using spaces. :)  I'll continue to look into this.  Sorry for the problem.

File: src/main/java/org/isf/dicom/manager/DicomManagerInterface.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/dicom/manager/DicomManagerInterface.java:36]: Done

File: src/main/java/org/isf/dicom/manager/DicomManagerInterface.java (hunk 3 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/dicom/manager/DicomManagerInterface.java:45]: Done.

File: src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperationRepository.java [hunk 4]
```diff
@@ -32,21 +32,21 @@
...
 
     @Modifying
     @Transactional
     @Query(value = "INSERT INTO MEDICALDSRWARD (MDSRWRD_WRD_ID_A, MDSRWRD_MDSR_ID, MDSRWRD_IN_QTI, MDSRWRD_OUT_QTI, MDSRWRD_LT_ID_A) " +
     		"VALUES (?, ?, ?, '0', ?)", nativeQuery= true)
-    public void insertMedicalWard(@Param("ward") String ward, @Param("medical") int medical, @Param("quantity") Double quantity, @Param("lot") String lot );
+    void insertMedicalWard(@Param("ward") String ward, @Param("medical") int medical, @Param("quantity") Double quantity, @Param("lot") String lot);
 
     @Query(value = "SELECT * FROM MEDICALDSRWARD WHERE MDSRWRD_WRD_ID_A = :ward", nativeQuery= true)
 	List<MedicalWard> findAllWhereWard(@Param("ward") char wardId);
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperationRepository.java:52]: Note that I didn't change this line but the indentation is off too.

**Comment by dbmalkovsky** [comment]
I am seriously thinking of closing this PR and trying it again fresh.

File: src/main/java/org/isf/visits/service/VisitsIoOperationRepository.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/visits/service/VisitsIoOperationRepository.java]: As I suspected some files seem to be a mix of space indentation and tab indentation (at least this one is).  In the following screenshot lines 15 through 18 use spaces and lines 20 through 27 use tab indentation. 
image

FWIW, the lines with spaces were added in this commit: 85c8f64e28ad645c9dc9b892749fcb16a2b6d32 on March 31, 2020

It appears that another standardizing pass like this [PR](https://github.com/informatici/openhospital-core/pull/50/files) is in order again.

Comment (dbmalkovsky) [reply src/main/java/org/isf/visits/service/VisitsIoOperationRepository.java]: I just ran a single command line based command that converted all spaces to tabs in all java files in the `develop` branch and 174 files were changed in `main` and 129 files were changed in `test`.

**Comment by dbmalkovsky** [comment]
I am going to open a new PR with these changes after applying the new formatting settings.

**Pull request closed without merging**